content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -433,6 +434,7 @@ menu, and select a category, or :guilabel:`Search More...` to find one that is n
    - `US Tax Compliance: Avatax elearning video
      <https://www.odoo.com/slides/slide/us-tax-compliance-avatax-2858?fullscreen=1>`_
 
+
```

Comment:
I believe only one space is needed between sections- I think this is an extra space.